db/migrate/20241108101517_create_rdvs.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateRdvs < ActiveRecord::Migration[7.0]
+  def change
+    create_table :rdvs do |t|
+      t.string :status
+      t.string :rdv_external_id
```

Comment:
a terme il nous faudra un index la dessus non ? (cf, les scopes booked et pending)

Reply:
not null

Reply:
ajouter la référence à l'instructeur qui a pris le rdv